### PR TITLE
Feature/server

### PR DESCRIPTION
**Pull request by lyssoi**

### CODE REVIEW

File: Server.cpp [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+const char* Server::KeventError::what() const throw(){
+	return ("Kevent Exception!!");
+}
+
+const char* Server::ClientAcceptError::what() const throw(){
+	return ("Client Accept Exception!!");
+}
+
+void	chage_events(std::vector<struct kevent> change_list, uintptr_t ident, int16_t filter, uint16_t flags)
```
Comment (Rillmo) [inline Server.cpp]: chage???

Comment (lyssoi) [reply Server.cpp]: 수정했습니다 ₩

**Review by minjacho42 — APPROVED** [review]
LGTM

**Review by Rillmo — APPROVED** [review]
LGTM

**Pull request merged**